### ISSUE TITLE
Does not start, can not find ./lib/start_server

### ISSUE THREAD

**Issue by kolis** [PolymerLabs/polyserve#52]
I just installed v 0.7.0
using `npm install -g polyserve`
but when starting the application I get the following error
```
module.js:340
    throw err;
    ^

Error: Cannot find module './lib/start_server'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:289:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/opt/local/lib/node_modules/polyserve/main.js:12:16)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
    at Module.require (module.js:366:17)
```

**Comment by ewbutterfield** [comment]
I've got the same issue. It looks like this was introduced with the port to TypeScript

**Comment by raghav135** [comment]
+1

**Comment by Sorondare** [comment]
Same issue...

**Comment by alodavide** [comment]
Same issues

**Comment by alodavide** [comment]
For a quick fix unistall polyserve and install the old one. npm install polyserve@0.6.0

**Comment by Mygod** [comment]
:+1: Reversing to 0.6.0 works for me.

**Comment by MeTaNoV** [comment]
same here... back to v0.6.0...

**Comment by ajjohnston** [comment]
It looks like the TypeScript files are not being compiled on install.

**Comment by justinfagnani** [comment]
This should be fixed in 0.7.1. Sorry I missed the notifications here.

**Comment by justinfagnani** [comment]
Please re-open if it's still happening.
